src/components/TopBar/TopBar.vue

```diff
@@ -356,6 +363,25 @@ export default {
 				}
 			},
 		},
+
+		// TODO replace with conversation.status === 'availability'
+		'conversation.name': {
```

Comment:
Hmm maybe it is better to fetch the absence status in `NewMessage` in userAbsence(). In that component, you have all the information about the conversation too. `TopBar` has nothing to do with it